pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -103,16 +103,9 @@ func (p RESTStorageProvider) PostStartHook() (string, genericapiserver.PostStart
 			_ = wait.PollImmediateUntil(
 				retryCreatingSuggestedSettingsInterval,
 				func() (bool, error) {
-					should, err := shouldEnsureSuggested(flowcontrolClientSet)
```

Comment:
there is probably a reason to have `shouldEnsureSuggested` in place. are there any risks if we `ensure` all objects both suggested and mandatory?

Reply:
Yes.  We explicitly and deliberately made a distinction between _required_ and _suggested_ configuration, and we explicitly and deliberately made the suggested configuration be something that the cluster operators can supplement, delete, and/or modify in any way they want.

Yes that means the operators have to think and potentially do something when migrating to a new release, in cases where they will want different configuration in the new release.  I do not think automatically smashing in the suggested config is a better solution.